libs/payments/legacy/src/lib/plan-mapper.util.spec.ts

```diff
@@ -24,7 +24,8 @@ describe('PlanMapperUtil', () => {
       defaultMapper = new PlanMapperUtil(
         defaultCommonContent,
         defaultPurchaseDetails,
-        stripeMetadata
+        stripeMetadata,
+        false
```

Comment:
Non-blocking, but would be good to have a test where contentfulEnabled is `true`.